dashboard/src/shared/Auth.test.ts

```diff
@@ -192,3 +192,15 @@ describe("Auth", () => {
     });
   });
 });
+
+describe("is403FromAuthProxy", () => {
+  it("does not assume to be authenticated from the auth proxy if the message contains a service account", () => {
+    expect(
+      Auth.is403FromAuthProxy({
+        status: 403,
+        data:
+          'namespaces is forbidden: User "system:serviceaccount:kubeapps:kubeapps-internal-kubeops" cannot list resource "namespaces" in API group "" at the cluster scope',
+      } as AxiosResponse<any>),
+    );
```

Comment:
What's the expectation here? There's no matcher following the `expect(...)`. Did you mean to use `.toBe(true)`?

Reply:
oops, thanks for catching, it should be `false`